lint/ruff.bzl

```diff
@@ -90,7 +90,7 @@ def ruff_fix(ctx, executable, srcs, config, patch):
         output = patch_cfg,
         content = json.encode({
             "linter": executable._ruff.path,
-            "args": ["check", "--fix"] + [s.path for s in srcs],
+            "args": ["check", "--quiet", "--fix"] + [s.path for s in srcs],
```

Comment:
doesn't this also throw away other output that we might want to show users, in the failure case? I think we only wanted to have `--quiet` on success